clients/CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ if(BUILD_CLIENTS_TESTS)
   if(NOT CONVERT_SOURCE)
     set(CONVERT_SOURCE ${CMAKE_SOURCE_DIR}/deps/convert.cpp)
   endif()
-  execute_process(COMMAND ${CMAKE_CXX_COMPILER} ${CONVERT_SOURCE} -O3 -o ${PROJECT_BINARY_DIR}/mtx2csr.exe RESULT_VARIABLE STATUS)
+  execute_process(COMMAND ${CMAKE_CXX_COMPILER} ${CONVERT_SOURCE} -O3 -Wl,--enable-new-dtags,--build-id=sha1,--rpath,$ENV{ROCM_EXE_RPATH} -o ${PROJECT_BINARY_DIR}/mtx2csr.exe RESULT_VARIABLE STATUS)
```

Comment:
This should not be needed.

Reply:
HI @ntrost57 
Updated the commit message. With frtlib-add-rpath opiton set in  rocm.cfg ,Clang compiler will add hard coded RUNPATH .Planning to remove the option from rocm.cfg while building stg2 components.
rocSPARSE was failing while executing the binary mtx2csr.exe from build directory